src/plopp/backends/matplotlib/image.py

```diff
@@ -75,13 +77,15 @@ def __init__(
         self,
         canvas: Canvas,
         data: sc.DataArray,
+        artist_number: int = 0,
         shading: str = 'auto',
         rasterized: bool = True,
         **kwargs,
     ):
         self._canvas = canvas
         self._ax = self._canvas.ax
         self._data = data
+        self._artist_number = artist_number
```

Comment:
Is this used anywhere?

Reply:
Not in the image, but the other artists (scatter, line) have one to keep track of the color to use.
I will ignore it in a better way.